io/net/infiniband/ib_pingpong.py

```diff
@@ -44,16 +44,24 @@ def setUp(self):
         '''
         To check and install dependencies for the test
         '''
+        local = LocalHost()
         interfaces = netifaces.interfaces()
+        device = self.params.get("interface", default=None)
+        if device in interfaces:
+            self.iface = device
+        elif local.validate_mac_addr(device) and device in local.get_all_hwaddr():
+            self.iface = local.get_interface_by_hwaddr(device).name
+        else:
+            self.iface = None
```

Comment:
@PavamanSubramaniyam I think we don't need this as we are cancelling the test any way.

Reply:
@Naresh-ibm we need this for teardown.. else the cancel fails abrubtly

Reply:
thanks